Sito/take_away.php

```diff
@@ -3,46 +3,101 @@
 
 	session_start();
 
-	$paginaHTML = file_get_contents('html/take_away.html');
-
-	$oggettoConnessione =  new DBAccess();
-
-	if ($oggettoConnessione->openDBConnection())
+	if (isset($_SESSION['username']) && isset($_POST['action']) && $_POST['action'] == 'aggiungi')

```

Comment:
Prima controllare se la c'è la sessione:
- in caso ci sia si controlla se c'è la post e si fa quel che si deve fare
- in caso non ci sia controllare se c'è la post, e se c'è ritornare qualche messaggio affinchè venga ricaricata la pagina